simplesat/sat/policy/undetermined_clause_policy.py

```diff
@@ -58,6 +58,11 @@ def _build_id_to_clauses(self, clauses):
         for c in clauses:
             for l in c.lits:
                 table[abs(l)].append(c)
+
+        # Make sure all installed packages appear in the table
+        for pid in self._installed_ids:
+            table[pid]
```

Comment:
I missed it was a defaultdict. The more I use them, the less I like them :(

That's a small function, so that's fine as is.